tools/topology/sof-hda-generic-idisp.m4

```diff
@@ -46,6 +47,18 @@ PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 	4, 3, 2, s32le,
 	48, 1000, 0, 0)
 
+# Low Latency playback pipeline 5 on PCM 4 using max 2 channels of s32le.
```

Comment:
comment should be capture instead of playback.

Reply:
thanks! I will correct it.